### DISCUSSION TITLE
Autostart

### DISCUSSION THREAD

**Discussion by PauLuke** [iberianpig/fusuma#274] [Q&A] [answered]
I follow the instructions to install fusuma in arch linux, but I don't find in any place how to autostart it every boot. There's a easy way to do that?

**Comment by iberianpig** [comment, marked as answer]
I think you should check how to autostart application for desktop environment you installed. https://wiki.archlinux.org/title/autostarting#On_desktop_environment_startup

 
Following link is about autostart for gnome https://github.com/iberianpig/fusuma#autostart-gnome-session-properties
